jboss/container/wildfly/galleon/provision-server/configure.sh

```diff
@@ -7,9 +7,9 @@ if [ ! -d "$GALLEON_DEFAULT_SERVER" ]; then
   exit 1
 fi
 
-if [ -f $JBOSS_CONTAINER_MAVEN_35_MODULE/scl-enable-maven ]; then
+if [ -f $JBOSS_CONTAINER_MAVEN_36_MODULE/scl-enable-maven ]; then
   # required to have maven enabled.
-  source $JBOSS_CONTAINER_MAVEN_35_MODULE/scl-enable-maven
+  source $JBOSS_CONTAINER_MAVEN_36_MODULE/scl-enable-maven
```

Comment:
For ubi7 SCL will likely still be there, but on 8, I'm not sure if this will be present. So it would be wise to build a test image using 8 / JDK 11 and see if maven is populated correctly in the environment (and on the path etc).